src/components/dialogs/payment.vue

```diff
@@ -62,39 +104,50 @@ import { ActionBindingIF } from '@/interfaces/store-interfaces'
 @Component({
   components: {
     RequestDetails,
-    FeeSummary
-  },
-  props: {
-    onActivate: {
-      type: Function,
-      default: async () => undefined
-    },
-    onCancel: {
-      type: Function,
-      default: async () => {}
-    }
```

Comment:
I replaced `onCancel` with a prop decorator/declaration below, and deleted `onActivate` which is no longer used.